src/HasAttributes.php

```
@@ -115,18 +121,20 @@ public function removeAttribute(string|UnitEnum $key): static
      * @param mixed $value Attribute value.
+     * @param string $prefix Prefix to ensure (for example, `aria-`, `data-`, `on`).
      *
      * @return static New instance with the updated `attributes` value.
      */
-    public function setAttribute(string|UnitEnum $key, mixed $value): static
+    public function setAttribute(string|UnitEnum $key, mixed $value, string $prefix = ''): static
     {
         $new = clone $this;
 
-        AttributeBag::set($new->attributes, $key, $value);
+        AttributeBag::set($new->attributes, $key, $value, $prefix);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\AttributeBag' in method 'setAttribute'. (undefined)

(StaticAccess)